src/components/forms/pool_actions/InvestForm/components/InvestPreviewModal/components/InvestSummary.vue

```diff
@@ -114,7 +131,19 @@ function weeklyYieldForAPR(apr: string): string {
         <div class="summary-table-label" v-text="$t('potentialWeeklyYield')" />
         <div class="summary-table-number">
           {{ fNum(totalWeeklyYield, currency) }}
-          <BalTooltip icon-size="sm" width="72" class="ml-2" noPad>
+          <BalTooltip icon-size="sm" width="72" noPad>
+            <template v-slot:activator>
+              <StarsIcon
+                v-if="props.pool.hasLiquidityMiningRewards || hasThirdPartyAPR"
+                class="h-4 text-yellow-300"
+              />
+              <BalIcon
+                v-else
+                name="info"
+                size="sm"
+                class="text-gray-300 ml-2"
```

Comment:
^ to be consistent with how we use "stars" and "info" icons.

Here's how it looks:

<img width="292" alt="Screen Shot 2021-12-15 at 3 47 04 pm" src="https://user-images.githubusercontent.com/254095/146125512-9131d9d7-778d-453e-8b2e-493e0db7a22a.png">


<img width="499" alt="Screen Shot 2021-12-15 at 3 46 45 pm" src="https://user-images.githubusercontent.com/254095/146125505-19f21e16-d391-4910-a414-d0f2e41c922c.png">